image/rhel/create-bundle.sh

```diff
@@ -23,6 +23,7 @@ extract_from_image() {
   [[ -n "$image" && -n "$src" && -n "$dst" ]] \
       || die "extract_from_image: <image> <src> <dst>"
 
+  mkdir -p "${dst}"
```

Comment:
Do we have a guarantee that `dst` is a folder and not file?
- If yes, then this is fine.
- If no, then we must add code that would prevent creating a directory that would be named exactly as the file. Illustration:

We want to do `cat /tmp/path/file`

```shell
$ touch file
$ cp file /tmp/path/file
cp: /tmp/path/file: No such file or directory
# cannot copy, because /tmp/path is absent

# we fix that as in this PR
$  mkdir -p /tmp/path/file
# and copy again
$ cp file /tmp/path/file
# now it worked, but what is the result?
$ ls /tmp/path/file
file
$  cat /tmp/path/file
cat: /tmp/path/file: Is a director
# ups, that is not what we wanted
```

Reply:
`dst` can be a file e.g. the `ldb` case in that script.